cmd/keeper/keeper.go

```diff
@@ -113,9 +131,6 @@ func (p *PostgresKeeper) getSUConnParams(keeperState *cluster.KeeperState, setPa
 		"dbname":           "postgres",
 		"sslmode":          "disable",
 	}
-	if setPassword {
-		cp["password"] = p.cfg.pgSUPassword
-	}
```

Comment:
Not sure why I addedt the setPassword bool but it's actually not needed... thanks.